app/packs/src/apps/mydb/elements/details/researchPlans/ResearchPlanMetadata.js

```diff
@@ -1,85 +1,77 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
+import { defaultsDeep } from 'lodash';
 import {
   Form, Button, Row, Col
 } from 'react-bootstrap';
 import ResearchPlansFetcher from 'src/fetchers/ResearchPlansFetcher';
 
 require('@citation-js/plugin-isbn');
 
+const emptyMetadata = {
+  title: '',
+  subject: '',
+  format: '',
+  version: '',
+  state: '',
+  url: '',
+  landing_page: '',
+  alternate_identifier: [],
+  related_identifier: [],
+  description: [],
+  dates: [],
+  geo_location: [],
+  funding_reference: []
+};
+
 export default class ResearchPlanMetadata extends Component {
   constructor(props) {
     super(props);
+    const { researchPlan } = props;
     this.state = {
-      researchPlan: {},
-      researchPlanMetadata: {
-        title: '',
-        subject: '',
-        format: '',
-        version: '',
-        state: '',
-        url: '',
-        landing_page: '',
-        alternate_identifier: [],
-        related_identifier: [],
-        description: [],
-        dates: [],
-        geo_location: [],
-        funding_reference: []
-      }
+      researchPlanMetadata: defaultsDeep({}, researchPlan.research_plan_metadata, emptyMetadata),
```

Comment:
I don't understand what you mean regarding the rename. The metadata state is taken from the passed in `researchPlan` prop instead of additionally passing it in as a prop. Do the names not match anywhere?

I'll check regarding lodash. I did not investigate what the server returns for metadata. So maybe we can get rid of setting defaults entirely.

Reply:
ok you are partly right. I misread this line here. But still, you can remove the property researchPlanMetadata from the proptype definition in line 504 :)